pkg/database/fake/database_generated.go

```diff
@@ -35,8 +36,8 @@ func CreateTestLogUrls() []LogUrlRow {
 }
 
 type LogUrlRow struct {
-	Uuid types.UID
-	Url  string
+	Uuid               types.UID
+	ContainerName, Url string
```

Comment:
This look strange to me.  Why not put them both on their own line with their own types?  Is this a go standard?  I know it's possible to do it this way, but do we want to?

Reply:
>> I know it's possible to do it this way, but do we want to?
> The question of if we want to do this stylistically is different.

Yes, that's what' I'm asking.

Reply:
I've added it to the agenda for Monday.

Reply:
moved `ContainerName` to its own line per discussion today